micro-slack/build.gradle

```diff
@@ -0,0 +1,11 @@
+description = 'micro-slack'
```

Comment:
So I cannot see  src/main/resources/META-INF.services Plugin file? Or a class that `implements Plugin`, are they missing from this PR?